purchases/src/main/kotlin/com/revenuecat/purchases/common/diagnostics/DiagnosticsTracker.kt

```diff
@@ -44,6 +46,15 @@ internal class DiagnosticsTracker(
         const val BILLING_DEBUG_MESSAGE = "billing_debug_message"
     }
 
+    private val commonProperties = if (appConfig.store == Store.PLAY_STORE) {
+        mapOf(
+            "play_store_version" to appConfig.playStoreVersionName,
+            "play_services_version" to appConfig.playServicesVersionName,
+        ).filterNotNullValues()
+    } else {
+        null
```

Comment:
Nit: since we're falling back to `emptyMap()` anyway, why not have it here so that it's cleaner where it's used?
```suggestion
        emptyMap()
```